### PR TITLE
Dark

### PR DESCRIPTION
**Pull request by btzr-io**
## Added
New theme: `Blueberry`

## Fixes
- Download percentage indicator overlay #1271
- Fix dark theme: #1034

### CODE REVIEW

**Comment by btzr-io** [comment]
Fix dark theme: https://github.com/lbryio/lbry-app/issues/1034

File: src/renderer/scss/_gui.scss [hunk 1]
```diff
@@ -69,7 +69,7 @@ ul {
 }
 
 input {
-  border-bottom: var(--input-border-size) dotted var(--input-border-color);
+  border-bottom: var(--input-border-size) solid var(--input-border-color);
```
Comment (seanyesmunt) [inline src/renderer/scss/_gui.scss]: The `dashed`/`dotted` should stay the same

**Comment by btzr-io** [comment]
@seanyesmunt I Added the new colors as an additional theme ->  `blueberry`, 
so now we have three themes: `light, dark and blueberry`.

**Comment by btzr-io** [comment]
@seanyesmunt  Ready for the next review

**Comment by tzarebczan** [comment]
@btzr-io not sure if we want to maintain 3 themes going forward, but I'll let @seanyesmunt make the call. 

Will give this a shot later tonight/tomorrow.

**Comment by btzr-io** [comment]
@seanyesmunt  Ok, it's done.

File: src/renderer/scss/_gui.scss [hunk 2]
```diff
@@ -92,10 +92,17 @@ input {
     background: var(--input-copyable-bg);
     color: var(--input-copyable-color);
     padding: 10px 16px;
-    border: 1px dashed var(--input-copyable-border);
+    border-bottom: 1px dashed var(--input-copyable-border);
```
Comment (seanyesmunt) [inline src/renderer/scss/_gui.scss]: This shouldn't change. It should be a border on every side

File: src/renderer/scss/_gui.scss [hunk 3]
```diff
@@ -92,10 +92,17 @@ input {
...
   }
 
-  &:not(.input-copyable):not(.wunderbar__input):not(:placeholder-shown) {
+  &:not(.input-copyable):not(.wunderbar__input):not(:placeholder-shown):not(:disabled) {
+    &:hover {
+      border-color: var(--input-hover-border-color);
+    }
+    &:active,
+    &:focus {
+      border-color: var(--color-primary) !important;
```
Comment (seanyesmunt) [inline src/renderer/scss/_gui.scss]: This isn't the correct style. We shouldn't need to change anything here.

File: src/renderer/scss/_gui.scss [hunk 4]
```diff
@@ -107,6 +114,7 @@ input {
 }
 
 input::placeholder {
+  color: var(--input-placeholder-color) !important;
```
Comment (seanyesmunt) [inline src/renderer/scss/_gui.scss]: We shouldn't need an `!important` here. If there is something that is overriding this style, we should remove that, or make this a more specific rule with classnames

File: src/renderer/scss/component/_modal.scss [hunk 5]
```diff
@@ -18,11 +18,13 @@
 }
 
 .modal {
+  /* Set modal buttons background color */
+  --btn-bg-alt: var(--modal-btn-bg-color);
```
Comment (seanyesmunt) [inline src/renderer/scss/component/_modal.scss]: Do we need this? It doesn't look like it's doing anything. We should set any variables in `vars.scss`

File: src/renderer/scss/component/_search.scss [hunk 6]
```diff
@@ -31,29 +31,37 @@
...
   &:focus {
-    background-color: var(--color-bg);
+    background-color: var(--search-active-bg-color);
     border-radius: 0;
-    border-bottom: 1px solid var(--color-grey);
+    border-bottom: 1px solid var(--color-divider);
     box-shadow: var(--box-shadow-button);
   }
+
+  &::-webkit-input-placeholder {
```
Comment (seanyesmunt) [inline src/renderer/scss/component/_search.scss]: Can we not use `::placeholder` here?

File: src/renderer/scss/component/_modal.scss [hunk 7]
```diff
@@ -31,6 +30,11 @@
   box-shadow: var(--box-shadow-layer);
   max-width: var(--modal-width);
   word-break: break-word;
+
+  .btn.btn--alt {
+    // Set modal buttons bg color
+    background-color: var(--modal-btn-bg-color);
+  }
```
Comment (btzr-io) [inline src/renderer/scss/component/_modal.scss]: @seanyesmunt Not sure if this is the right way to do this, but I don't have time to add a new class to all the `btn-alt` used inside the modal component :stuck_out_tongue:

File: src/renderer/scss/_gui.scss (hunk 2 above)
Comment (btzr-io) [reply src/renderer/scss/_gui.scss]: Not sure why I did that :upside_down_face:

File: src/renderer/scss/component/_modal.scss (hunk 7 above)
Comment (seanyesmunt) [reply src/renderer/scss/component/_modal.scss]: I think this is ok for now.

Comment (seanyesmunt) [reply src/renderer/scss/component/_modal.scss]: Well, do modals in dark mode not use the regular alt button style?

**Comment by seanyesmunt** [comment]
I totally messed up the number of commits. Could you squash?

I think the only two issues I still see are that we lost the background-color for the `text-copyable` used in the address and file selector. And the "You have {lbc}" doesn't change color on hover.

File: src/renderer/scss/component/_modal.scss (hunk 7 above)
Comment (btzr-io) [reply src/renderer/scss/component/_modal.scss]: @seanyesmunt yes but it's the same color as the background of the modal :upside_down_face:,
see https://design.lbry.io

**Comment by seanyesmunt** [comment]
Ahh gotcha

On Apr 16, 2018 9:45 PM, "Baltazar Gomez" <notifications@github.com> wrote:

*@btzr-io* commented on this pull request.
------------------------------

In src/renderer/scss/component/_modal.scss
<https://github.com/lbryio/lbry-app/pull/1269#discussion_r181932250>:

> @@ -31,6 +30,11 @@
   box-shadow: var(--box-shadow-layer);
   max-width: var(--modal-width);
   word-break: break-word;
+
+  .btn.btn--alt {
+    // Set modal buttons bg color
+    background-color: var(--modal-btn-bg-color);
+  }

@seanyesmunt <https://github.com/seanyesmunt> yes but it's the same color
as the background of the modal 🙃,
see https://design.lbry.io

—
You are receiving this because you were mentioned.

Reply to this email directly, view it on GitHub
<https://github.com/lbryio/lbry-app/pull/1269#discussion_r181932250>, or mute
the thread

**Comment by btzr-io** [comment]
Fix conflicts: a00f4eb

**Pull request merged**